arbitrator/arbutil/src/evm/mod.rs

```diff
@@ -10,6 +10,9 @@ pub mod user;
 // params.SstoreSentryGasEIP2200
 pub const SSTORE_SENTRY_GAS: u64 = 2300;
 
+// params.WarmStorageReadCostEIP2929
+pub const TRANSIENT_BYTES32_GAS: u64 = 100;
```

Comment:
Let's call it `TRANSIENT_OP_GAS` or similar

Reply:
:+1: